c/validation/adbc_validation_util.cc

```diff
@@ -141,7 +141,25 @@ int MakeSchema(struct ArrowSchema* schema, const std::vector<SchemaField>& field
   CHECK_ERRNO(ArrowSchemaSetTypeStruct(schema, fields.size()));
   size_t i = 0;
   for (const SchemaField& field : fields) {
-    CHECK_ERRNO(ArrowSchemaSetType(schema->children[i], field.type));
+    switch (field.type) {
+    case NANOARROW_TYPE_DECIMAL128:
+      // TODO: how can we avoid hard-coding precision and scale?
```

Comment:
Need to still find a better way to specify precision / scale than hard-coding here, but I think that will be dependent on larger test design